templates/includes/sidebar.html

```diff
@@ -1,4 +1,11 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li><a href="{% url "taxi:driver-detail" pk=user.id %}">User: {{ user.username }}</a></li>
```

Comment:
The URL tag for `taxi:driver-detail` uses the `user.id` as the primary key. Ensure that the `Driver` model's primary key corresponds to the `user.id`. If the `Driver` model uses a different field as the primary key, this could lead to incorrect URL generation.